src/components/navbar.jsx

```diff
@@ -1,6 +1,17 @@
 import { Link } from "react-router-dom";
+import { useCart, useWishlist } from "../context";
+import "../App.css";
 
 export const Navbar = () => {
+    const token = localStorage.getItem("token");
+    const { cartState } = useCart();
+    const { wishlistState } = useWishlist();
+    const { cartProductsCount } = cartState;
+    const { wishlistProductsCount } = wishlistState;
+
+    const logoutHandler = () => {
+        localStorage.setItem("token","")
```

Comment:
You can use `localStorage.removeItem("token")`

Reply:
Modified. Thanks